frontend/common/src/components/AppInsights/AppInsightsContext.tsx

```diff
@@ -0,0 +1,16 @@
+import React, { createContext } from "react";
+import { reactPlugin } from "./ReactPlugin";
+
+const AppInsightsContext = createContext(reactPlugin);
+
+const AppInsightsContextProvider = ({
+  children,
```

Comment:
Is this context provider being used anywhere?  Shouldn't it be used in our context sandwich like the locale provider and client provider?